submissions/Eugene-CG/DOM/style.css

```diff
@@ -0,0 +1,97 @@
+:root {
+  --full-size: 100%;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.delegate-events-container {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+}
+.wrapper {
+  min-height: 100vh;
+  max-width: var(--full-size);
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  grid-template-columns: 1fr 2fr 2fr;
+  grid-template-areas:
+    "header header header"
+    "main main main"
+    "footer footer footer";
+  background-image: url(./img/gitlogoforme.jpeg);
+  color: rgb(255, 255, 255);
+}
+.popup {
+  display: none;
+}
+.header {
+  display: flex;
+  justify-content: space-around;
+  grid-area: header;
+  line-height: 4;
+  letter-spacing: 0.25rem;
+}
+ul {
+  list-style: none;
+}
+input[type="checkbox"] {
+  height: var(--full-size);
+  width: var(--full-size);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  outline: none;
+}
+.menu {
+  height: var(--full-size);
+  width: 40%;
+  margin-left: 1rem;
+}
+.popup-btn-label {
+  display: none;
+}
+.menu > ul {
```

Comment:
Avoid styling by HTML tag. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Try to avoid creating too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).